<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatRoomUI</class>
 <widget class="QWidget" name="ChatRoomUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat Room</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="currentRoomNameLabel">
     <property name="text">
      <string>현재 방</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="chatTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="messageLineEdit">
       <property name="placeholderText">
        <string>메시지를 입력하세요...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="text">
        <string>전송</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="backToListButton">
     <property name="text">
      <string>목록으로 돌아가기</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
